### DISCUSSION TITLE
Copilot chat hallucinating and inventing contents of an entire non-existent private Github repo!

### DISCUSSION THREAD

**Discussion by talkingnews** [community/community#59447] [Copilot Conversations]
### Select Topic Area

Bug

### Body

Waited **so long** to get Copilot chat, and was excited to start my first attempts today. 

It appears all is not well, as I start out with a freeform prompt and then just see what happens if I accept what it suggests I do.

The thing is... **the repo I mention doesn't exist**! If I change the name to "audio-helpers", it does the same, but for audio formats.
I tried this after I pointed it at a repo that DID exist, and it told me completely wrong info, which seemed based on the name, not the code.

I'd understand if it said "`based on the name the code might do the following`", but it specifically says "`these are the ones explicitly mentioned *in the code*`". The code that doesn't exist.

This is worrying - what's happening here, and why, and how can I resolve it?
Note: The repo I'm trying to examine is private, but the same account that Copilot is linked to. If I try one that isn't, it tells me it can't examine the repo as it doesn't exist. Weird.

__________________________________

Me _

Can you examine the code repository at https://github.com/my-private-repo/video-helpers ?_

Copilot: 

`Yes, I can examine the code repository at https://github.com/my-private-repo/video-helpers. What would you like me to do specifically?`

_What are some examples of video formats that the video-helpers repository supports?_

```
Based on the code in the repository, the video-helpers library supports the following video formats:

MP4
MOV
AVI
WMV
It's possible that other formats are also supported, but these are the ones explicitly mentioned in the code.
```

_What methods are available in the VideoPlayer class of the video-helpers library?_

```
The VideoPlayer class in the video-helpers library provides the following methods:

playFromFile(file: string): Promise<void>: Plays a video file from the specified file path.
playFromUrl(url: string): Promise<void>: Plays a video file from the specified URL.
pause(): void: Pauses the currently playing video file.
```

__________________________________

**Comment by Akash1134** [comment]
Hi @talkingnews ,

Thanks for reporting and sharing your valuable feedback.

We have shared it with our team internally to look into.
